CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
     Bug #3623: Fix HiDPI on Windows
     Bug #4540: Rain delay when exiting water
     Bug #4701: PrisonMarker record is not hardcoded like other markers
+    Bug #4714: Crash upon game load in the repair menu while the "Your repair failed!" message is active
```

Comment:
Is this something we need to worry about with 0.45?

Reply:
Same with Bug 4715?

Reply:
I encountered the crash before but it only occurs when you're savescumming and you'll always have a save to fall back on when reopening the game, so it wouldn't be a blocker. It's also not a recent regression.

Reply:
OK